### ISSUE TITLE
Can you provide google drive link for pretrained models please?

### ISSUE THREAD

**Issue by tysnow2022** [Feobi1999/TDD#2]

**Comment by Feobi1999** [comment]
Ok, I will provide the google link in this week. Btw, the data and environment  preparation will be updated this weekend.

**Comment by tysnow2022** [comment]
Oh thankyou glad to hear that. I was struggling by my own. actually for the pretrained model a friend with Baidu account helped me. When i try to evaluate it i got the following issue.  I attached the screen shoot of the error 
and for full log information here is the link. 
https://drive.google.com/file/d/1avtwHQV9TQoRA-n7vOOANsvGkdoITVzW/view?usp=sharing
![log1](https://user-images.githubusercontent.com/24326843/168598101-73674d90-a19f-4da0-891c-403f817f8208.PNG)

**Comment by Feobi1999** [comment]
@tysnow2022 When I deleted some redundant code, I deleted a line of content by mistake.  I just fixed it and tested it, you can pull the latest version code. Sorry for the inconvenience.

**Comment by tysnow2022** [comment]
thank you the testing works but the training throws this error.
![error3](https://user-images.githubusercontent.com/24326843/168631317-c8b0afc5-5a87-4ed0-a302-70d1c49faa98.png)

**Comment by Feobi1999** [comment]
The training works well in my experiment server. Mention now we only support the one gpu one pair images strategy, which has been introduced in the Readme file.

**Comment by tysnow2022** [comment]
I am replying so lately. sorry for the inconvenience. I set the IMG_PER_BATCH_LABEL and  IMG_PER_BATCH_UNLABEL both to 1 in the config yaml file . and it starts the training but it shows the loss is 'nan' like this   "total_loss: nan,  loss_cls: nan  loss_box_reg: nan loss_cls_head_2: nan"
